.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  job:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        node: ['10', '12', '14']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install packages
+        run: yarn --no-progress --non-interactive --no-lockfile
```

Comment:
Could we use the cache here, and perhaps  `--frozen-lockfile --prefer-offline` instead of these flags?

Yarn should already be non-interactive in a CI environment.